### PR TITLE
REL-3859: Add the new GMOS B480 gratings to the ITC

### PR DESCRIPTION
**Pull request by andrewwstephens**

### CODE REVIEW

File: bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public StageModeNorth parse(String name, StageModeNorth defaultValue) {
         },
         B600_G5307(  "B600_G5307",  "B600",  600, Ictd.track("B600")),
         R600_G5304(  "R600_G5304",  "R600",  600, Ictd.track("R600")),
-        G480_G5309(  "B480_G5309",  "B480",  480, Ictd.track("B480")),
+        B480_G5309(  "B480_G5309",  "B480",  480, Ictd.track("B480")),
```
Comment (andrewwstephens) [inline bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java:81]: Look at this bit carefully - could this change affect other things?

Comment (swalker2m) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java:81]: It's fine.  Thanks for spotting this.  I overlooked it when we switched from G to B.

**Review by swalker2m — APPROVED** [review]
I think this looks fine.  I'll let Sebastian or Carlos merge though since I don't really track the ITC.

**Comment by sraaphorst** [comment]
Thanks, @cquiroz!

**Pull request merged**